build.gradle

```diff
@@ -33,6 +33,7 @@ dependencies {
   testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
   testCompile 'commons-io:commons-io:2.5'
   testRuntime 'ch.qos.logback:logback-classic:1.1.3'
+  testRuntime 'cglib:cglib-nodep:2.2'
```

Comment:
Here is the error:

> com.hierynomus.msdfsc.DFSTest
> should resolve dfs for a path FAILED
    org.spockframework.mock.CannotCreateMockException: Cannot create mock for class com.hierynomus.smbj.SMBClient. Mocking of non-interface types requires the CGLIB library. Please put cglib-nodep-2.2 or higher on the class path.